mkdocs/docs/HPC/conda.md

```diff
@@ -0,0 +1,36 @@
+# Conda
+
+We do not recommend the use of conda environments on the HPC clusters. 
+This section explains why and highlights some common issues.
+
+## Performance and Optimization
+
+Conda's packages are pre-compiled binaries that are designed to work on a wide range of systems. 
+This means they are not optimized for the specific architecture of HPC clusters, 
+leading to potential performance drawbacks compared to modules compiled specifically for the HPC environment.
+
+HPC modules on the other hand, are compiled for the specific architecture of the cluster,
+and are optimized for performance.
+
+## Compatibility and Dependency
+
+Using Conda in conjunction with centrally installed modules can lead to conflicts and unexpected errors, 
+making it difficult to debug and manage dependencies.
+
+## Package Availability
+
+Conda has a smaller repository of available packages compared to PyPI, the repository used by pip. 
+This can limit the availability of specific tools or libraries needed for certain workflows.
+
+## Environment and Installation Issues
+
+### Home Directory Usage
+
+Conda installs packages and environments in the user's home directory, 
```

Comment:
We have to give examples how one should instruct conda to put the cache and env directories to somewhere else than home, for example to $VSC_DATA

Reply:
added in [a7aa31a](https://github.com/hpcugent/vsc_user_docs/pull/649/commits/a7aa31a0488ac50a6ef2c0c1905e2a650d6dca74)